docs/reference/get-started/get-started-local-dev.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[get-started-elasticsearch-local-dev]]
+== Get started with {es} in local development
+++++
+<titleabbrev>Local dev (Docker)</titleabbrev>
+++++
+
+// Comment
+We present two options to get started by running {es} in Docker:
+
+* <<get-started-elasticsearch-local-dev-elasticsearch, Run {es} in Docker with security disabled>>
+* <<get-started-elasticsearch-local-dev-kibana, Run {es} and {kib} in Docker with security enabled>>
```

Comment:
How would we feel about switching the order of these here, and in the sections below?

People will gravitate towards the first option, and I think that should be the full install with security because:
1. Security is important, and we put a lot of effort into turning on security by default so that users don't get caught be ransomware attacks, etc. We're setting them up for pain if they run without security and don't understand the risks
2. Kibana is important. Most users should have Kibana and enabling Kibana later is harder
3. Getting started without security may be quicker, but converting over to using security later is more work than just doing it up front. Is users realise they need security, then they're going to be annoyed at how hard it is to switch (we put all our effort into making it easy up front, not easy to enable later)